IoT-Sample/Swift/IoTSampleSwift/ConnectionViewController.swift

```diff
@@ -19,207 +19,273 @@ import AWSMobileClient
 
 class ConnectionViewController: UIViewController, UITextViewDelegate {
 
-    @IBOutlet weak var activityIndicatorView: UIActivityIndicatorView!
-    @IBOutlet weak var logTextView: UITextView!
-
-    @objc var connected = false;
-    @objc var publishViewController : UIViewController!;
-    @objc var subscribeViewController : UIViewController!;
-    @objc var configurationViewController : UIViewController!;
-
-    @objc var iotDataManager: AWSIoTDataManager!;
-    @objc var iotManager: AWSIoTManager!;
+    let IOT_CERT = "IoT Cert"
+    let IOT_WEBSOCKET = "IoT Websocket"
+
+    var connectIoTDataWebSocket: UIButton!
+    var activityIndicatorView: UIActivityIndicatorView!
+    var logTextView: UITextView!
+    var connectButton: UIButton!
+    
+    @objc var connected = false
+    @objc var publishViewController : UIViewController!
+    @objc var subscribeViewController : UIViewController!
+    @objc var configurationViewController : UIViewController!
+
+    @objc var iotDataManager: AWSIoTDataManager!
+    @objc var iotManager: AWSIoTManager!
     @objc var iot: AWSIoT!
 
-    @IBAction func connectButtonPressed(_ sender: UIButton) {
+    func mqttEventCallbackWebsocket( _ status: AWSIoTMQTTStatus ) {
+        if case .connected = status {
```

Comment:
Prefer `guard/let` to big `if/else` blocks, especially when the `else` could reasonably be considered an early-exit condition

Reply:
I consider guard/let as a mechanism which is used to describe a precondition for executing the rest of the function.  In this case( yes, case -- ha ha),  we leveraging the existing function mqttEventCallback for all cases except cases.

This will probably go away once we separate control plane and data plane into separate view controllers anyways.

Not going to update htis.

Reply:
Updated this to two completely separate functions to provide clarity.